funcx_endpoint/funcx_endpoint/endpoint/results_ack.py

```diff
@@ -81,6 +81,9 @@ def persist(self):
     def load(self):
         """ Load unacked results from disk
         """
-        if os.path.exists(self.data_path):
-            with open(self.data_path, 'rb') as fp:
-                self.unacked_results = pickle.load(fp)
+        try:
+            if os.path.exists(self.data_path):
+                with open(self.data_path, 'rb') as fp:
+                    self.unacked_results = pickle.load(fp)
+        except pickle.UnpicklingError:
+            logger.warning(f"[WARNING] Cached results {self.data_path} appear to be corrupt. Proceeding without loading cached results")
```

Comment:
```suggestion
            logger.warning(f"Cached results {self.data_path} appear to be corrupt. Proceeding without loading cached results")
```

If the log format contains the level, then showing "WARNING" as part of the string is redundant.